hieunc-guestbook-dojo/static/js/guestbook/models/Greeting.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Created by hieunc on 25/11/2015.
+ */
+
+define([
```

Comment:
- config pycharm dùng indent là TAB nhé http://192.168.1.248/redmine/projects/rakumo/wiki/Pycharm_config
- format lại toàn bộ file js/html/css dùng indent là TAB